### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_Screenreader:Voiceover does not read the content written under treeview.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52395]
**Environment Details:**
VSCode Version : 1.24.1
OS Version :MAS1.3.1,MACOS High Sierra 10.13.5 Beta (17F70a)
**Additional Details:**
Screenreader,Voiceover
**Repro Steps:**
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to tree view "Installed" and "Recommendation" under the search extension.
4. Check what Voiceover is reading.

**Actual:**
Voiceover does not read the  content written under the treeview.

**Expected:**

Voiceover should read the  content which is present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not able to know which content is written on the screen.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference**

[[Accessibility]A11y_VSCode_ViewExtensions_ScreenReader_Voiceover.pptx](https://github.com/Microsoft/vscode/files/2118065/Accessibility.A11y_VSCode_ViewExtensions_ScreenReader_Voiceover.pptx)

**Comment by JacksonKearl** [comment]
\duplicate #51968
